src/components/cqlLibraryEditor/CqlLibraryEditor.tsx

```diff
@@ -63,20 +63,20 @@ const CqlLibraryEditor = ({
   const terminologyServiceApi = useTerminologyServiceApi();
   const [valuesetMsg, setValuesetMsg] = useState(null);
   const [valuesetSuccess, setValuesetSuccess] = useState(true);
+  const [isLoggedInToUMLS, setIsLoggedInToUMLS] = useState<boolean>(false);
 
   const updateElmAnnotations = async (cql: string): Promise<ElmTranslation> => {
     if (handleClick) {
       if (cql && cql.trim().length > 0) {
         const data = await elmTranslationServiceApi.translateCqlToElm(cql);
 
         let valuesetsErrors = null;
-        const tgt = getTgt();
-        const tgtValue = getTgtValue(tgt);
+        const isLoggedIn = await Promise.resolve(checkLogin());
```

Comment:
I'm wondering if `Promise.resolve` is redundant here with the `async/await`.

Reply:
@jkotanchik-SB Yes it's needed here otherwise we won't get the right value.